content/docs/en-us/whats-new.md

```diff
@@ -1,6 +1,31 @@
 ---
 title: What's New in 2.6
 ---
+
+## New in 2.6.3
```

Comment:
The heading levels are the same for the version as well as the children - we should use a different heading style.